tests/unit/test_llm_providers.py

```
@@ -0,0 +1,79 @@
+from unittest.mock import MagicMock
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)